protected/migrations/m170928_220602_create_sequence_data_table.php

```diff
@@ -0,0 +1,59 @@
+<?php
+
+class m170928_220602_create_sequence_data_table extends OEMigration
+{
+    public function up()
+    {
+        // Source: http://www.microshell.com/database/mysql/emulating-nextval-function-to-get-sequence-in-mysql/
+
+        $this->createTable('sequence_data', array(
+                'sequence_name' => 'varchar(100) NOT NULL',
+                'sequence_increment' => 'int(11) unsigned NOT NULL DEFAULT 1',
+                'sequence_min_value' => 'int(11) unsigned NOT NULL DEFAULT 1',
+                'sequence_max_value' => 'bigint(20) unsigned NOT NULL DEFAULT 18446744073709551615',
+                'sequence_cur_value' => 'bigint(20) unsigned DEFAULT 1',
+                'sequence_cycle' => 'boolean NOT NULL DEFAULT FALSE',
+            )
+        );
+
+        $this->addPrimaryKey('sequence_data_pk', 'sequence_data', 'sequence_name');
+
+        $stmt = <<<EOL
```

Comment:
EOSQL would probably make more sense as the heredoc delimiter.

Reply:
Fixed